packages/cli/cloud/package.json

```diff
@@ -47,6 +47,7 @@
   "dependencies": {
     "@strapi/utils": "5.4.0",
     "axios": "1.7.4",
+    "boxen": "5.1.2",
```

Comment:
nice one! do you confirm it's ok to add more modules? not sure what are the rules of this repo ^^'